src/core/src/core_logic/PatchInstaller.py

```diff
@@ -20,6 +20,8 @@
 import time
 from core.src.bootstrap.Constants import Constants
 from core.src.core_logic.Stopwatch import Stopwatch
+from core.src.package_managers import AptitudePackageManager
```

Comment:
note that'll need to be resolved -- shouldn't be importing this here given class hierarchies and dependencies. This is an indication that something needs to change in code organization.

Reply:
This is stale entry now. Will remove this. I have changed way to check if package_manager is apt.